frontend/src/pages/projects/screens/detail/storage/StorageTable.tsx

```diff
@@ -26,6 +26,7 @@ const StorageTable: React.FC<StorageTableProps> = ({ pvcs, refresh, onAddPVC })
   const storageTableData: StorageTableData[] = pvcs.map((pvc) => ({
     pvc,
     storageClass: storageClasses.find((sc) => sc.metadata.name === pvc.spec.storageClassName),
+    accessModes: pvc.spec.accessModes,
```

Comment:
There is no need to take the `accessModes` out here. See the `pvc` above, you can always get the data from `pvc` directly. So you can remove this line here.